iaso/api/completeness_stats.py

```diff
@@ -349,54 +355,86 @@ def to_dict(row_ou: OrgUnitWithFormStat):
                 "org_unit": row_ou.as_dict_for_completeness_stats(),
                 "form_stats": row_ou.form_stats,
                 "org_unit_type": row_ou.org_unit_type.as_dict_for_completeness_stats() if row_ou.org_unit_type else {},
-                "parent_org_unit": row_ou.parent.as_dict_for_completeness_stats() if row_ou.parent else None,
+                "parent_org_unit": row_ou.parent.as_dict_for_completeness_stats_with_parent()
+                if row_ou.parent
+                else None,
+                "has_children": has_children(row_ou),
             }
 
+        def to_map(row_ou: OrgUnitWithFormStat):
```

Comment:
It was maybe not clear but all these methods could have been moved out of the class ... It doesn't really matter and I'll approve it